### ISSUE TITLE
Constraints issue

### ISSUE THREAD

**Issue by borut-t** [onfido/onfido-ios-sdk#78]
### What was the expected behaviour?

No constraints issue

### What happened instead?

Reported constraints issue in console

### Version info:

- Cocoapods (if applicable): 1.6
- Onfido SDK (Debug/Release): 10.6.0
- iOS: 12.1.4
- Xcode: 10.1
- Device/Simulator: iPhone
- Device/Simulator language: en

### Integration configuration:

```swift
    do {
      let config = try OnfidoConfig.builder()
        .withToken(Constants.Configuration.Onfido.sdkKey)
        .withApplicantId(applicant.id)
        .withDocumentStep(ofType: .drivingLicence, andCountryCode: "us")
        .build()
      let flow = OnfidoFlow(withConfiguration: config)
      return IdentityCardConfig(applicant: applicant, flow: flow)
    } catch {
      throw error
    }
```

### Steps to reproduce:

after scanning front screen of drivers license, I get this
```
2019-03-28 11:08:53.637528+0100 MyApp PRODUCTION[1401:111878] [LayoutConstraints] Unable to simultaneously satisfy constraints.
	Probably at least one of the constraints in the following list is one you don't want. 
	Try this: 
		(1) look at each constraint and try to figure out which you don't expect; 
		(2) find the code that added the unwanted constraint or constraints and fix it. 
(
    "<NSLayoutConstraint:0x28336ecb0 UIView:0x111d9c520.top == Onfido.OverlayView:0x1069575a0.top + 131.532   (active)>",
    "<NSLayoutConstraint:0x283089b80 UIView:0x111d9c520.top == Onfido.OverlayView:0x1069575a0.top + 145.139   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x283089b80 UIView:0x111d9c520.top == Onfido.OverlayView:0x1069575a0.top + 145.139   (active)>

Make a symbolic breakpoint at UIViewAlertForUnsatisfiableConstraints to catch this in the debugger.
The methods in the UIConstraintBasedLayoutDebugging category on UIView listed in <UIKitCore/UIView.h> may also be helpful.
2019-03-28 11:08:53.640454+0100 MyApp PRODUCTION[1401:111878] [LayoutConstraints] Unable to simultaneously satisfy constraints.
	Probably at least one of the constraints in the following list is one you don't want. 
	Try this: 
		(1) look at each constraint and try to figure out which you don't expect; 
		(2) find the code that added the unwanted constraint or constraints and fix it. 
(
    "<NSLayoutConstraint:0x2830846e0 UIView:0x1069eaa20.top == Onfido.OverlayView:0x111da06e0.top + 145.139   (active)>",
    "<NSLayoutConstraint:0x2830a5590 UIView:0x1069eaa20.top == Onfido.OverlayView:0x111da06e0.top + 176.691   (active)>"
)
```

**Comment by anuragajwani** [comment]
Hi @borut-t,

Thanks for reaching out to us. We will look into the issue and get back to you as soon as we can.

Anurag

**Comment by anuragajwani** [comment]
Hi @borut-t,

Sorry for the late reply. We have the task to fix this on our backlog.

Anurag

**Comment by keremgund** [comment]
Hi @borut-t ,
First of all sorry for the late response. 
This issue has been resolved in 14.0.0 version. 

I'm closing the ticket, please open new one if you think issue persists.

Thanks,
Kerem
